TAC/Projects/Enarx/enarx-proposal.md

```diff
@@ -0,0 +1,66 @@
+   1.  Enarx came out of some conversations between Nathaniel McCallum and Mike Bursell (the project’s founders) and led to them presenting the idea to a number of people at Red Hat, who agreed to provide some initial resources to see if it was plausible.  When the Consortium was announced, we realised very quickly that it was very well aligned.
+
+1. statement on alignment with Consortium charter's mission
+   1.  Enarx aims to be hardware-vendor neutral, providing a run-time environment which works across platforms.  We hope to make it easy for people to deploy their own applications or applications created by 3rd party vendors, supporting the “protecting data in use” part of the charter which is core to the Consortium.  Being conceived as an open source project, we welcome input and collaboration from all parties.  Minimal Trusted Compute Base and high auditability are high on our list of design principles. The code is very modular, meaning that there may well be opportunities for other projects to re-use components for their needs. 
+
+1. link to *current* Code of Conduct (if one is adopted already)
+   1.  TBD. 
+
+1. sponsor from TAC, if identified (a sponsor helps mentor projects)
+   1.  Mike Bursell (Red Hat)
+
+1. project license 
+   1.  Apache 2.0
+
+1. source control (GitHub by default)
+   1.  GitHub
+
+1. issue tracker (GitHub by default)
+   1.  GitHub
+
+1. external software dependencies (including licenses)
+   1.  TBD
+
+1. release methodology and mechanics
+   1.  TBD
+
+1. names of initial committers, if different from those submitting proposal
+   1.  Nathaniel McCallum, Mike Bursell
+
+1.  briefly describe the project's leadership team and decision-making process
+   1.  Currently run with Nathaniel McCallum coordinating code contributions and Mike Bursell leading most other project-related matters, but both adopting an “Open Management” style with much delegation and discussion on key architectural and project matters.  We plan to revisit the governance and leadership model once we have a larger number of contributors.
+
+1. link to any documented governance practices
+   1.  n/a
+
+1. preferred maturity level (see stages below)
+   1.  Awaiting clarification on stage naming
+
+1. list of project's official communication channels (slack, irc, mailing lists)
+   1.  #enarx on freenode
+   1.  Issue tracking on GitHub
+
+1. link to project's website 
+   1.  https://enarx.io
+
+1. links to social media accounts
+   1.  @enarxproject (Twitter)
+
+1. existing financial sponsorship
+   1.  Support from Red Hat
```

Comment:
Support from Profian

Reply:
Updated and pushed. I added Profian as a new line, not replacing Red Hat.